<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BedlamToolsClass</class>
 <widget class="QMainWindow" name="BedlamToolsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BedlamTools</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>580</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_img">
     <attribute name="title">
      <string>BIN</string>
     </attribute>
     <widget class="QGraphicsView" name="graphics">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>510</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="button_update">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>IMG:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_img">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Additional offset:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_img_offset">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_x">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_y">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>131</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="button_open_img">
         <property name="text">
          <string>Open BIN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_bin">
         <property name="text">
          <string>No File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_open_pal">
         <property name="text">
          <string>Open PAL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pal">
         <property name="text">
          <string>No File</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLabel" name="img_count">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="img_header">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>IMG Header</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>IMG Count</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>RAW</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bedlamtools.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBox_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BedlamToolsClass</receiver>
   <slot>update_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BedlamToolsClass</receiver>
   <slot>update_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>604</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_img_offset</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BedlamToolsClass</receiver>
   <slot>update_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_img</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BedlamToolsClass</receiver>
   <slot>update_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_image()</slot>
 </slots>
</ui>
